ui/nuxeo-user-group-management/nuxeo-group-management.js

```diff
@@ -556,9 +556,7 @@ import './nuxeo-user-group-permissions-table.js';
     }
 
     _hasAdministrationPermissions(user) {
-      return (
-        user && (user.isAdministrator || (this.isMember(user, 'powerusers') && this.groupname !== 'administrators'))
-      );
+      return this.hasAdministrationPermissions(user) && this.groupname !== 'administrators';
```

Comment:
If `this.groupname` equals `administrators` then it will always return false and `user` won't be able to proceed even if `user.isAdministrator` is true